drupal/code/profiles/kadaprofile/kadaprofile.install

```diff
@@ -676,3 +676,25 @@ function kadaprofile_update_7123() {
   $enable_dependencies = TRUE;
   module_enable($modules, $enable_dependencies);
 }
+
+/**
+ * Implements hook_update().
+ *
+ * Enables context error module
+ */
+function kadaprofile_update_7124() {
```

Comment:
Missing the related info file dependency. IMO context_error should be a dependency of pori_error_pages, which should be enabled in the configuration feature instead of here.